catalog/imaging/imaging-plugin-nitf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -15,16 +15,20 @@
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
     <OCD name="Nitf Imaging Input Transformer"
-         id="NITF_Input_Transformer">
+         id="NITF_Thumbnail_Plugin">
         <AD
                 description="Maximum length of longest side of NITF overview image in pixels. The input transformer will calculate the size of the shorter side so that the overview will have the same aspect ratio as the original."
                 name="Overview image maximum side length (pixels)" id="maxSideLength" required="true"
                 type="Integer" default="1024"/>
+        <AD
+                description="Maximum NITF file size in Megabytes to create thumbnails for. Creating thumbnails for large nitfs can cause out-of-memory exceptions."
+                name="Maximum NITF file size to process in Megabytes" id="maxNitfSizeMB" required="true"
+                type="Integer" default="10"/>
 
     </OCD>
 
-    <Designate pid="NITF_Input_Transformer">
-        <Object ocdref="NITF_Input_Transformer"/>
+    <Designate pid="NITF_Thumbnail_Plugin">
+        <Object ocdref="NITF_Thumbnail_Plugin"/>
```

Comment:
I'd rename this to NITF_Render_plugin or something as well.  This may be a good time to add booleans for "Create Overview" and "Store Original Image".